### ISSUE TITLE
Simple Mobile Tools bought by ZipoApps?

### ISSUE THREAD

**Issue by inson1** [SimpleMobileTools/General-Discussion#241]
https://www.reddit.com/r/SimpleMobileTools/comments/187w64x/simple_mobile_tools_bought_by_zipoapps/

**Comment by Aga-C** [comment]
What's going with this @tibbi @naveensingh ?

**Comment by naveensingh** [comment]
I can confirm it's true, but I don't have details on who, when, or where.

**Comment by tibbi** [comment]
ye, the sale is in process

**Comment by Aga-C** [comment]
As a regular contributor I'm very disappointed.

**Comment by mijorus** [comment]
Understandable, I'm sorry you couldn't find a suitable business model for this project. Thanks for your work in these years

**Comment by 0verEngineer** [comment]
I hope ZipoApps understands that they cannot simply make the apps closed source and suck out the userbase because these apps are used by people that value the FOSS principles.
Otherwise the users will switch to real FOSS alternatives and/or the apps will be forked.

**Comment by tswistak** [comment]
> I hope ZipoApps understands that they cannot simply make the apps closed source

That would violate the current license of all the apps. GPLv3 forbids it.

**Comment by moneytoo** [comment]
> That would violate the current license of all the apps. GPLv3 forbids it.

The owner of the code still owns the copyright and can relicense the code under whatever license he/she wishes.

**Comment by tswistak** [comment]
> The owner of the code still owns the copyright and can relicense the code under whatever license he/she wishes.

That's not as easy. There are a few things:

- https://www.gnu.org/licenses/gpl-faq.html.en#Consider 
- https://www.gnu.org/licenses/gpl-faq.html.en#GPLInProprietarySystem (if apps use any external GPL libraries)
- https://opensource.stackexchange.com/questions/270/is-it-possible-to-close-an-open-source-project

**Comment by ColorfulRhino** [comment]
Like many others, I am very disappointed about this move 😞

Not directly because of a sale, I can understand that you want money for your work. But because of **who** the apps are sold to.
This company ZipoApps seems to be mainly about quantity instead of quality, has lots of ad-ridden crapware in the PlayStore and does not adhere to privacy standards. All in all, most values of this company seem to completely contradict the values of SimpleMobile apps.

**Comment by cyrneko** [comment]
Another app lost to ZipoApps. They *will* add advertisements, trackers and more as they have done for other well-made apps that they've purchased in the past.

**Comment by afarah1** [comment]
Thank you for maintaining it thus far.

Since it's GPL I suppose anyone interested in maintaining it can simply fork and republish.

Until then, I disabled auto-update on PlayStore for the SimpleMobileTools apps I have.

**Comment by naveensingh** [comment]
Of course, the license allows the project to be forked and maintained and I would be happy to do so in my free time but I believe it's essential to honor @tibbi's decision. He's been tirelessly maintaining this project since it was created, it would only be fair if he gets to determine its future direction. I'll proceed with maintaining a fork only upon receiving a green light from @tibbi.

**Comment by BuZZ-dEE** [comment]
@naveensingh What do you mean by: "it would only be fair if he gets to determine its future direction."? The direction of the fork?

Why do you need green light from @tibbi ?

**Comment by naveensingh** [comment]
When I mentioned 'it would only be fair if he gets to determine its future direction' I was referring to the direction of the current project, not some forked version. As for seeking a green light from @tibbi, it's a courtesy to ensure that he's comfortable with the idea of someone else taking on the maintenance. Given his experience and commitment to the project, I want to make sure he approves before proceeding with the fork.

**Comment by M00NJ** [comment]
> The owner of the code still owns the copyright and can relicense the code under whatever license he/she wishes.

No, relicencing is very complicated. It requires the consent of everyone who ever contributed, which will not be given I believe. So the owner can sell, but the licence has to stay the same. They are allowed to relicense everything they have contributed though, so they could change the licence of some parts of the code, but relicensing everything is a violation of the terms.

**Comment by inson1** [comment]
@naveensingh I think he lost the right when he sold it and killed it. And he already showed what he wants.

**Comment by BuZZ-dEE** [comment]
https://github.com/you-apps

https://www.reddit.com/r/SimpleMobileTools/comments/18929pq/simplemobiletools_was_sold_alternatives/

**Comment by ryosoftware** [comment]
Is  time to move to another project. Unfortunately is the life cycle of all apps: sell or die when the devels move to other projects or centers their hearts in the real life.

**Comment by MarsInsanity** [comment]
I'm sorry, I get that you would have to sell Simple Mobile Tools, but to ZipoApps? Hell no, hell to the back no.

They're notorious for buying apps from developers like TreyDev and then packing the apps with ads, putting a "no-ads" subscription to remove it for $15usd a week.

![Screenshot_20231203-040100_Speedometer~2](https://github.com/SimpleMobileTools/General-Discussion/assets/53840971/6f33f564-cb19-4f5c-821e-dcd73c8f6d48)
^^ An example of an app published by ZipoApps, (This app should only have **one** $2-10 lifetime purchase to remove ads, not a weekly subscription model more expensive than a month of Spotify)

As far as I know, they go directly against the ethos of Simple Mobile Tools, and they're going to alienate your user base in their pursuit to make as much money as possible. What they're going to do with your set of apps is going to be so predictable that it's almoat comical, and I would never use an app published by them.

**Comment by Acters** [comment]
I hate the fact that these people think 15/week, ~65/month, is a great price for removal of ads and "premium support" when they suck at making updates and their apps need little to no support at all. Its all subterfuge to pretend that its a great service when its literally a nothing burger with gold shavings sprinkled on top. Jerks and a giant waste of space on the internet, and the developers not announcing the fact they are selling the apps are just as big of jerks for being non-transparent. 

Time to edit all my posts to boycott these apps and stop them from hurting our communities.

**Comment by 0verEngineer** [comment]
If someone will fork it i will definetly help and contribute.

@naveensingh I think you should just do it if you really want, you should not push the dev into "allowing" it because this is a shady company..

I have no experience in Kotlin but i want to learn it anyway so...

**Comment by inson1** [comment]
I dont think we should also fork everything 
- its too much work 
- there are better (for some simple apps) alternatives anyway

(for example the calculator)

**Comment by cordelia04m** [comment]
(((Zipoapps)))

**Comment by musq** [comment]
It's always a controversial step and even though I too hate this sale, **I support the original author for this decision**.

On a personal level, it's hard to maintain open source software without any significant financial support. Even if everything goes right, there could be lots of nuances going on in his life at a personal level which **we don't need to know**, that could be driving this decision.

Since Simple Mobile Tools is a [Gold Sponsor of Linux Mint](https://www.linuxmint.com/sponsors.php), I feel the situation must be dire for him to take such a step now.

The beauty of FOSS software lies in the fact that anybody can fork and continue the maintenance. We don't need to expect or rely on the original author to support the software in perpetuity 🙂.

I believe the best we can do here is to tell him we are unhappy about this, but we still respect his decision and his personal space. Thank you @tibbi for creating and maintaining these awesome apps all these years!

**Comment by txort** [comment]
So, your typical user that is currently using google play simple mobile apps will just randomly one day get ads on apps they installed and possibly some personal data mined via telemetry? Am I missing something?

**Comment by inson1** [comment]
@musq 

> It's always a controversial step and even though I too hate this sale, I support the original author for this decision.
>
> On a personal level, it's hard to maintain open source software without any significant financial support. Even if everything goes right, there could be lots of nuances going on in his life at a personal level which we don't need to know, that could be driving this decision.
> 
> Since Simple Mobile Tools is a [Gold Sponsor of Linux Mint](https://www.linuxmint.com/sponsors.php), I feel the situation must be dire for him to take such a step now.


I dont respect that decision. If he said what was happening and some good reason for that, before this happened then I could and would respect that decision. Maybe he would even get that money from us if we knew what would happened without that.

**Comment by shuvashish76** [comment]
![image](https://github.com/SimpleMobileTools/General-Discussion/assets/48226312/aefb98b9-3f38-42cb-b896-32d0fef33ddd)
The [official website](https://www.simplemobiletools.com/) PlayStore dev ID linked to : https://play.google.com/store/apps/dev?id=9070296388022589266 (Simple Mobile **Tools**) with 0 apps. [[archives](https://archive.is/https://play.google.com/store/apps/dev?id=9070296388022589266*)]

While all apps repository badges linked to : https://play.google.com/store/apps/dev?id=8214346176194980263 (Simple Mobile **Tool**). Here no "developer website" provided in _Additional Information_. [[archives](https://archive.is/https://play.google.com/store/apps/dev?id=8214346176194980263*)]
What's going on?

**Comment by jmtwelfth** [comment]
> The [official website](https://www.simplemobiletools.com/) PlayStore dev ID linked to : https://play.google.com/store/apps/dev?id=9070296388022589266 (Simple Mobile **Tools**) with 0 apps. [[archives](https://archive.is/https://play.google.com/store/apps/dev?id=9070296388022589266*)]
> 
> While all apps repository badges linked to : https://play.google.com/store/apps/dev?id=8214346176194980263 (Simple Mobile **Tool**). Here no "developer website" provided in _Additional Information_. [[archives](https://archive.is/https://play.google.com/store/apps/dev?id=8214346176194980263*)] What's going on?

The Developer (tibbi) renamed his 'developer/business' name for his apps pending sale to ZipoApps.

I presume he renamed the developer account that holds the suite of apps so he can retain the name "Simple Mobile Tools"  on another as he owns the domain name.

Going through all the apps on his developer page on Google Play, SMS Messenger, Calendar, Flashlight, Contacts, Gallery, Draw, File Manager, Camera, Calculator, Keyboard, Notes now all have the "contains ads" warning.

A major breach of trust considering his own website says, and I quote:

"A group of simple, open source Android apps without ads and unnecessary permissions, with customizable colors."

In other words: He sold his installed userbase to another company.

**Comment by ch3thanhs** [comment]
So, will the people using these apps get a notification or any prompt inside that app informing them of the sale/ads/telemetry?

**Comment by DeltaTangoLima** [comment]
Well, I've already forked and mirrored all the repos to my own Forgejo server, plus grabbed the latest APKs. If enough people do that same, we can rebuild from here.

**Comment by shuvashish76** [comment]
> So, will the people using these apps get a notification or any prompt inside that app informing them of the sale/ads/telemetry?

Good luck to those who think [ZipoApps](https://zipoapps.com/) care their users.
> "A group of simple, open source Android apps **without ads and unnecessary permissions**, with customizable colors."

**Comment by TOGLK** [comment]
Wow, you sold out your user base and fed them all to the lions, even bringing them into the lions enclosure.

I'll be actively avoiding anything you develop going forwards, thanks for your deception, you've ruined my trust in you for life, good bye, good riddance.

**Comment by naveensingh** [comment]
Alright, I have decided I'll maintain a fork.

**Comment by x0apr** [comment]
I'm just a noob, but has anyone else noticed the growing trend where some entities begin projects as open source only to later close them?

Yes, the maintainers and project owners dedicate a significantly greater amount of time compared to other contributors but still, I think it's not appropriate
Nevertheless, I appreciate SimpleMobileTools; it was wonderful while it lasted.

**Comment by zunjae** [comment]
> I'm just a noob, but has anyone else noticed the growing trend where some entities begin projects as open source only to later close them?
> 
> 
> 
> Yes, the maintainers and project owners dedicate a significantly greater amount of time compared to other contributors but still, I think it's not appropriate
> 
> Nevertheless, I appreciate SimpleMobileTools; it was wonderful while it lasted.

Can you name a few examples?

**Comment by HaneulCheong** [comment]
I see people trying to defend or at least understand this decision, and I would probably do the same thing in normal situations, cuz your project, your choice.

However, I also strongly believe that this could have been communicated better in advance, especially before someone opens an issue specifically questioning what is going on regarding third parties.

Regardless of whether they were about to announce this soon, let alone announce it at all, *the fact that we are realizing this through a user submitted issue makes the whole practice very shady*, which is why I believe the user base right now has every right to complain about it.

**Comment by HaneulCheong** [comment]
@inson1 

> I dont respect that decision. If he said what was happening and some good reason for that, before this happened then I could and would respect that decision. Maybe he would even get that money from us if we knew what would happened without that.

Exactly, well said.

**Comment by shuvashish76** [comment]
https://zipoapps.com/ ; "Better apps for a better world"
> We are ZipoApps, we acquire the best apps and take them to the next level.

So they acquire "best" apps & then make them "better", interesting.

**Comment by matkoniecz** [comment]
Note that F-droid has known safe versions that will not become ad infested. In general, F-droid is a good place for filtered apps. Even if no maintained fork will appear then at least app will not start showing porn/scam or pornscam ads.

**Comment by matkoniecz** [comment]
> So they acquire "best" apps & then make them "better", interesting.

noone said that "take them to the next level" needs to be better. Especially without saying next level of what :(

**Comment by Vendicated** [comment]
this is really sad to hear. i guess this essentially means the death of SimpleMobileTools

however it is also understandable. maintaining all these apps is a crazy effort and im sure the donations were way too low for how much effort was spent (as is the case for most of free software sadly 😓)

thank you a lot for making and maintaining these apps for all this time ❤️ i really hope someone continues these projects via fork

**Comment by pvagner** [comment]
Now reading this blog post feels really very very ironic https://www.simplemobiletools.com/blog/how-much-does-your-data-cost-we-know-the-price

**Comment by C8opmBM** [comment]
> Now reading this blog post feels really very very ironic https://www.simplemobiletools.com/blog/how-much-does-your-data-cost-we-know-the-price

Wow, just wow! Written in Aug/Sept current year.

**Comment by shuvashish76** [comment]
It goes against their all [memes](https://t.me/s/simplemobiletools), blog posts & principles for which SMT known for.

> Selling your app takes three easy steps:
> 1. Initial talk and negotiation within 10 days of first contact
> 2. Evaluation of your app within 25 days
> 3. Offer-to-buy received within 10 days
> However.... Most of our deals are closed within 14 days!

Source: https://zipoapps.com/

It takes only 14days to change your core principles.

**Comment by its-me-ct** [comment]
While the current version of the code will keep working, this is not the most important part of the sale. I'd assume ZipoApps cares more about the brand recognition and the millions of auto-updating installs on unsuspecting users' devices.

While Simple Mobile Tools were pretty good apps, the only thing that made them great was the independence from large corpos and the reassurance that personal data will not be harvested and sold. On my devices it's not a big deal, I'm exclusively using apps from f-droid anyway, but I also installed these apps on less tech-savy relatives' devices for the above stated reasons.

I hope Zipo paid you enough to make it all worth it. My condolences to all the voluntary contributors whose work was sold against their best interest.

**Comment by Cyanic76** [comment]
> I'm just a noob, but has anyone else noticed the growing trend where some entities begin projects as open source only to later close them?

Isn't that just Microsoft's EEE principle?

**Comment by x011** [comment]
Fork and start a new project, this one is dead.

**Comment by tibbi** [comment]
Right, I know that it is a really controversial step that upset many users, sadly the quality of the whole Android ecosystem is dropping really quickly recently and I wanted to avoid slow death. Thanks a lot for the support that me and the apps received over the years, it means a lot to me :)

Anyway, @naveensingh has been a developer for a long time so Im sure he will be able to take the project further.

**Comment by M-Reimer** [comment]
Seems reasonable. To avoid slow death, just kill the apps right away 🫣

**Comment by dengste** [comment]
Hi @tibbi, first off: Thank you for all the hard work over the years you put into these apps. I've used them for a long time, and I'm grateful for that. And yes, I donated in the past, but make no mistake: I'm not thinking I'm entitled to anything I get basically for free.

That being said, your lack of communication is troubling. I think a few question need to be answered, best in a public blog post from you.

* Will the apps remain FOSS, or is a change of license planned?
* Which releases will be affected? What about the Pro-versions people paid for, and what about the versions on F-Droid?
* Will apps be silently updated for people and suddenly contain trackers/adware/etc., or will the old apps be discarded and published under a new name?

**Comment by tibbi** [comment]
Not sure about the future plans, they likely wont be open source. But that means that the f-droid variants will stay the same as now.

**Comment by dengste** [comment]
Are you aware that anyone who contributed to your apps need to agree to a change of license?

**Comment by tibbi** [comment]
not really, thats not how it works

**Comment by dengste** [comment]
I'm afraid that's exactly how copyright works. Unless you had people sign a CLA and have copyright over all the contributed code, you cannot unilaterally change the license, unless either everyone agrees to a license change, or you remove/rewrite all the contributed code. I strongly recommend you get a lawyer before you sign something with promises you cannot actually keep.

**Comment by x011** [comment]
I have cloned all repositories and downloaded all applications.
After today, any apps created using these repos will be profit-oriented.
FOSS died here.
I would advise everyone not to update any of these app from now on.

**Comment by x011** [comment]
> Are you aware that anyone who contributed to your apps need to agree to a change of license?

You're absolutely right. I don't think @tibbi is aware of this, but I also think, and please correct me if I'm wrong, that any licence change that happens **after** the sale is complete, isn't his problem/responsability.

**Comment by vyashole** [comment]
@naveensingh I think @tibbi 's last comment points to the fact that they are OK with you forking it. 


I have cloned all the smt repos and if @naveensingh decides not to fork it, I'll try. I'm not sure if I can stand up to @tibbi and @naveensingh

**Comment by tibbi** [comment]
discuss any future changes with @naveensingh , I wont be around it anymore.

**Comment by x011** [comment]
> discuss any future changes with @naveensingh , I wont be around it anymore.

Please don't spoil all the good work you've done so far with this kind of irresponsible attitude. You owe the community an explanation.

**Comment by tibbi** [comment]
well, as I wrote, I just dont like where Android is heading anymore. Android 14 is the worst version so far (Ive been around since Android 1.6), but the newly redesigned Dev console app is also just too bad now and it would take me a lot more extra time to maintain the apps, reviews etc. The general UX is getting worse for both users and devs for quite a few years now.

**Comment by ghost** [comment]
You still haven't addressed your complete lack of communication leading up to this, and your comments replying to this issue have missed the mark entirely.

This situation would have been much better for everyone had you been transparent from the beginning.
As it is now, you've gone completely against your principles, blog posts, social media posts etc posted just a few weeks ago and sold to the most infamous company on the app store. You've lost all your dignity.
Hopefully this will be a learning experience for you.

**Comment by naveensingh** [comment]
For those who are interested: 

**New Github organization:** https://github.com/FossifyX

I'll be **transferring** all existing forks on my account to this new organization. To stay updated, either star/watch the repositories you are interested in (currently on my account) or follow the organization I linked above which will ultimately be the new home for all simple apps (I'm optimistic).

General project discussions, questions, ideas, etc. can be continued here: https://github.com/FossifyX/General-Discussion 

Also, now that all is said and done, can we please move forward? For those of you who are interested and would like to continue the discussion related to the forked project, please visit the link above. For those of you who don't want to do anything with simple apps anymore, please look for alternatives (check the Reddit thread by @inson1) or disable updates and you'll be able to use the apps until future Android updates break them completely.

**Comment by marek22k** [comment]
> The owner of the code still owns the copyright and can relicense the code under whatever license he/she wishes.

Don't forget the contributors' changes (multiple owners). All contributors must also agree.

**Comment by LambdAurora** [comment]
> not really, thats not how it works

First of all, lemme prepend this by saying that I Am Not A Lawyer, meaning I may miss a detail and be wrong.

I have looked around and I was not able to find a CLA or similar, meaning that any contributions to this project has not resulted in any of the contributors waving their rights to the legal owner of the project. Though, if there was indeed a CLA to sign before contributing and it stated that you waved your rights, then the following paragraphs won't be true.

Once the sale is complete, if zipoapps decides to go closed source *and* keep the external contributions, any contributors will have the ability to claim license infringement and ask zipoapps to comply under the terms of the license: either by removing every of your contributions (depending on how big of a contributor you are this might be a big blow to them), or by forcing them to keep the project open-source as defined by the GPLv3 license.

If zipoapps do make those apps closed source, and you're a contributor for whom it is uncomfortable, I'd recommend to consult a lawyer first to inform yourself of what you can do and the proper steps to take.
Though, there's also the question of is it worth it to go after them? Maybe energy is better spent working on the fork, given who this project panders to, I don't think zipoapps will inherit much of the userbase if they go down the bad road.

While I am mostly an external observer in this story, I wish a good continuation to the original author of this project, I can see how bailing out can become a desirable option when things get bad. I do find it weird it has been chosen to sell the project instead of giving it to another maintainer who could have continued the project's legacy. I guess there's some personal things that pushed to the sale, so I wouldn't hold it too much against the author. I also wish a good continuation to the fork as well.

**Comment by VishalNehra** [comment]
So sad to see the decision by the dev here. But indirectly this is the result of users, just taking open source apps for granted, thinking if it's open source it should be free. 
I've personally dealt with the heat of adding a trial period in one of our open source apps at team amaze. I'm personally not doing it for money, but I expect if anyone is contributing to one of our projects, I should be able to reward them with some sort of monetary benefit. I've also been getting dozens of offers for aquisition for so many years but that goes against the spirit of open source.
Anyways, like everyone has mentioned previously. Selling this isn't going to be as simple, you'll have to get waiver from anyone who's even contributed a single line of code, or rewrite the whole thing.
Applause to @naveensingh for taking this further 👏👏

**Comment by Ponchale** [comment]
@tibbi Hello, a pleasure Alfonso, we are in charge of the Midori browser oriented towards privacy, security and before selling, please let's talk.

**Comment by mvmike** [comment]
I don't care about the reasons why the app was sold, if Zipo will keep anything open source (they won't) or if it's legal to re-license (it isn't), the thing is what's been sold is not the app but its google play users.

Most of us here know how to stick to old versions or use f-droid, but the vast majority of others don't, what will happen is that they will start using an ad-infested piece of crap after unknowingly updating the app, even though they installed something that was open source and with complete opposite core principles. The way this has (not) been communicated also says a lot about how shady this move is.

You should feel ashamed, @tibbi, and honestly fuck companies like Zipo that use these kind of predatory practices.

**Comment by Handrail9** [comment]
Rest in Piss to the people who bought the Simple Phone.
https://simplephone.tech/eu/

**Comment by esjarc** [comment]
@mvmike Even worse, people who recommended these apps to family and friends now look stupid.

They have to tell everyone to uninstall these apps as soon as possible and need to assist with migrations. Even worse if someone used in-app storage to save their contacts, like some privacy-oriented websites and communities recommend.

It's beyond disrespectful to anyone who supported the project – e. g. by contributing code or translations, donating, recommending these apps to others – to sell the user base without telling them in advance (blog post, in-app notification, etc.).

**Comment by LearningAsIGo71** [comment]
It looks like tibbi was between a rock and hard place: Hating the direction Android is going and unable/unwilling to spend the increasing time required to properly maintain all the apps, but prior to abandoning the project, needing a way to be reimbursed for some of the thousands of hours devoted to the project without compensation (yes I know there was certainly some compensation in the form of donations, but I doubt anywhere near minimum wage for all the hours devoted). 

So what would you do in suh a situation? While the apps are in a decent stage of maintenance I would approach some potential buyers known for respecting users' privacy and see what type of offers I get without before disclosing anything publicly, for fear a bunch of people start bashing the product, one star reviewing it, and/or uploading various forks (competitors) of it to Google Play, thus diminishing the value of the SMT apps to potential buyers. 

With that said, if the highest bidder was a notoriously bad, greedy, anti-FOSS, data harvesting company, I would certainly NOT go with the highest bidder, first on principal, and second to protect my userbase from the predatory, data-harvesting practices of the company. 

Ah well, live and learn tibbi. Hopefully your next venture will prove to be more satisfying, rewarding, while maintaining utmost integrity and moral principals.

**Comment by CryptGoat** [comment]
Just wanted to drop by to say thanks to @tibbi for putting all the hard work you have put into developing these apps.

And shame on you @tibbi for selling out your users and customers to the opposite site of what have always promised. I absolutely understand that you might want to earn a living and selling the apps to another entity (while complying to the GPLv3) is totally fine but this is just a huge disappointment.

**Comment by dengste** [comment]
@tibbi I really mean this with the best intentions when I tell you: from a purely legal standpoint, you most probably cannot sell these apps. Please, go ask a lawyer who knows about copyright and FLOSS (these are hard to come by, but if you are genuinely interested, I can recommend people you can talk to, as I have worked quite a bit in FLOSS license compliance). 
In a nutshell: the day you accepted your first pull request that added copyrightable code (which usually means roughly more than 10 lines), you lost single ownership of the code for these apps. Just go in one of your repos and run

`git ls-files | xargs -n1 git blame --line-porcelain | sed -n 's/^author //p' | sort -f | uniq -ic | sort -nr`

Most of the names you see there with more than 10 lines still have copyright on the code they have contributed. You cannot sell this code, it is simply not yours. You would need explicit permission to do that from all contributors, or you need to remove their contributions.

**Comment by roomnoom** [comment]
You don't owe anyone anything, you are just trying to make a living without being miserable like the rest of us. Thank you for your work so far, I'll simply migrate to another app and not insult you for not working on something you don't enjoy anymore just for my own benefit like a narcissistic manchild. Wishing you the best in your future endeavors.

**Comment by CryptGoat** [comment]
> You don't owe anyone anything [...]

He absolutely DOES owe to pay attention who has contributed code to his apps. I would't mind the sale too much if he would comply to the GPL and if he hadn't chosen to sell out the privacy of his customers and users.

**Comment by dengste** [comment]
> You're absolutely right. I don't think @tibbi is aware of this, but I also think, and please correct me if I'm wrong, that any licence change that happens **after** the sale is complete, isn't his problem/responsability.

It is absolutely his responsibility if he signed a contract which claims that he has sole ownership of these apps, because in that case, ZipoApps can rightfully say that they assumed they completely own the code and hence can change the license.

**Comment by roomnoom** [comment]
> He absolutely DOES owe to pay attention who has contributed code to his apps. I would't mind the sale too much if he would comply to the GPL and if he hadn't chosen to sell out the privacy of his customers and users.

Sounds like an easy court case win then, let me know how it goes

**Comment by M00NJ** [comment]
> not really, thats not how it works

I'm kinda shocked that you are not aware of the terms of the licence your projects are protected by. No matter how they are going to relicence parts of the codebase or fork them, making the projects closed source is a clear violation of the terms. The whole point of this licence is to prevent that from happening. But I guess it's not your problem after all. It's the buyers problem and I'm sure they are aware of it, don't care and will do it anyways. But If someone is willing to spent time and resources on this, they could let the whole transaction fail because it's not quite legal what they seem to be planing and if they can't make them closed source, they might not want to buy them after all. Just making you aware...

**Comment by ch3thanhs** [comment]
> You don't owe anyone anything, you are just trying to make a living without being miserable like the rest of us. Thank you for your work so far, I'll simply migrate to another app and not insult you for not working on something you don't enjoy anymore just for my own benefit like a narcissistic manchild. Wishing you the best in your future endeavors.

![thats-bait.gif](https://github.com/SimpleMobileTools/General-Discussion/assets/76928501/c74a0bb7-8f43-4d5b-a00b-96c5717657f5)

**Comment by freshgum-bubbles** [comment]
Is there any resolution as to the GPL issues raised in the thread?
If there isn't a CLA, you're not legally allowed to relicense GPL code.

This is quite a pressing issue, as it means that the new owner can't
just move the apps into a private repository and add their usual
array of bloatware, adtech, etc.  Instead, it would need to be open-source.

**Comment by roomnoom** [comment]
> ![thats-bait.gif]

You're right he owes you everything, let me know how the court case goes since this is illegal and he's not just hurting your feelings by not maintaining your favourite apps until death

**Comment by CryptGoat** [comment]
> You're right he owes you everything, let me know how the court case goes since this is illegal and he's not just hurting your feelings by not maintaining your favourite apps until death

You know exactly that this is not what this is about.

**Comment by roomnoom** [comment]
> You know exactly that this is not what this is about.

So sue him then, or is complaining in github comments until you get bored all you're capable of?

**Comment by freshgum-bubbles** [comment]
It would be great if we could avoid (ego-driven) flamebait and focus on the actual issues :/

**Comment by cybersphinx** [comment]
> > not really, thats not how it works
> 
> I'm kinda shocked that you are not aware of the terms of the licence your projects are protected by. No matter how they are going to relicence parts of the codebase or fork them, making the projects closed source is a clear violation of the terms. The whole point of this licence is to prevent that from happening. But I guess it's not your problem after all.

It might be, if he signs a contract than contains a "I own all necessary rights to sell this code" clause and that isn't true.

**Comment by M00NJ** [comment]
> It might be, if he signs a contract than contains a "I own all necessary rights to sell this code" clause and that isn't true.

You're totally right. That would mean all sorts of trouble.

**Comment by ghost** [comment]
Does this potentially open up Tibor Kaputa to lawsuits from other contributors of code to the SimpleMobileTools suite, assuming he sells their code without their consent, and if he does not have a license agreement allowing him the rights to do so?

Without these rights, he would only be able to legally sell his own self-written code to ZipoApps?

This is a massive disrespect to everyone who has ever donated or contributed.

**Comment by topcaser** [comment]
Hope you are getting rich with this move @tibbi

**Comment by Quackdoc** [comment]
> Does this potentially open up Tibor Kaputa to lawsuits from other contributors of code to the SimpleMobileTools suite, assuming he sells their code without their consent, and if he does not have a license agreement allowing him the rights to do so?
>
> Without these rights, he would only be able to legally sell his own self-written code to ZipoApps?
>
> This is a massive disrespect to everyone who has ever donated or contributed.

He is allowed to sell the management and his code. which benefit of doubt lets assume he has done, Zipo has two options, either keep everything open source and just develop in house with code releases, Or remove all code that 3rd parties have contributed. 

IF all of the code was sold as if he had the right to it, then yes, that opens up copyright violations and whatnot. IF zipo modified and distributes the code as is, that opens them up to the same can of worms. Any contributor who has code in the distributed apps can file a DMCA if the code is not opensource

**Comment by Ponchale** [comment]
Astian has forked all the tools with the firm promise to always keep the source open source and without selling the products, we also promise not to place advertising or sell user data. https://github.com/goastian.

We invite everyone to join the Astian Suite project

**Comment by tswistak** [comment]
I have my hopes in @naveensingh's fork (https://github.com/FossifyX) as he was a long-time contributor to SMT, so I can be sure that he will continue the apps with old principles.

**Comment by jeannekamikaze** [comment]
Hi @tibbi, respect for your work and SimpleMobileTools, but I think (IANAL) you're getting into murky water here and risking yourself and ZipoApps an interesting lawsuit.

The GPL protects source code more than liberal licences do. In particular, unless contributors have transferred copyright ownership of their contributions to you, you cannot unilaterally decide to re-license the source code and make  it proprietary. This may be different for liberal licences, but the GPL specifically protects source code to a greater extent. Also, whatever proprietary adware ZipoApps wants to add is not compatible with the GPL. Therefore, unless you go and strip out every single third-party contribution to the source, the legal basis for re-licensing looks bleak.

I think the real question is what do you think should change to make future development of the apps sustainable. Are you simply looking for more funding, or have you given up altogether? These apps are great and used by every single GrapheneOS user that I know. I think everyone here wants their development to continue.

**Comment by vijay-prema** [comment]
@naveensingh and other contributors, if you are unhappy about your copyrighted code being closed source and abused by the new owner, you could try sending a copyright infringement notice to Google Play store to take the app down.  Easy way to do something if you want to, rather than wasting effort with lawyers.

IMO the only real value to the new owner is the Google Play Store account as it allows them to make a quick return by pushing ads/spyware to millions of unsuspecting users.

I have seen this before where a FOSS owner will sell control of the FF/Chrome extension store account or Google Play store account and the sketchy owner just push ads out on the store.  If anything, by taking down, you save the community of users.

Anyway, onwards and upwards.  Put a donation link once the new fork stabilizes.

**Comment by floopered** [comment]
> Not sure about the future plans, they likely wont be open source. But that means that the f-droid variants will stay the same as now.

Translation: "gpl violations inbound but thats not my problem any more"
Its funny to see how little you care about a detail that (hopefully) will sink the entire sale, leaving you with no acquisition and a completely tanked reputation

**Comment by LearningAsIGo71** [comment]
> > @floopered "Its funny to see how little you care about a detail that (hopefully) will sink the entire sale, leaving you with no acquisition and a completely tanked reputation"

Unless you contributed large sums of money to SMT, I don't see how you can justify such a harsh response.  Tibbi has likely devoted thousands of hours for years on these apps and you hope he gets a completely tanked rep? Furthermore, the apps have been open-source from the beginning and they are still available for no charge on F-Droid. Not only that, but Naveem, one of the project's contributors has forked all the apps (see above) and has committed to maintaining them. How about you? 
[Edit: confusing sentence]

**Comment by ghost** [comment]
> > > @floopered "Its funny to see how little you care about a detail that (hopefully) will sink the entire sale, leaving you with no acquisition and a completely tanked reputation"
> 
> Unless you contributed large sums of money to SMT, I don't see how you can justify such a harsh response. Tibbi has likely devoted thousands of hours for years on these apps and you hope he gets a completely tanked rep? Furthermore, the apps have been open-source from the beginning and they are still available for no charge on F-Droid. Not only that, but Naveem, one of the project's contributors has forked all the apps (see above) and has committed to maintaining them. How about you? [Edit: confusing sentence]

The reason there is such a harsh response isn't because we don't appreciate the hours they have put in, it is because the community was bait-and-switched with no warning, if this was just discussed beforehand, it wouldn't have been as huge of a deal as it is.

At current, it appears to be a sellout because of the way that it was done, with 0 input from the community or contributors.

**Comment by LearningAsIGo71** [comment]
> @tibbi Hello, a pleasure Alfonso, we are in charge of the Midori browser oriented towards privacy, security and before selling, please let's talk.

You are in charge of an abandoned app? Midori hasn't been updated on F-Droid in 2 years! To maintain proper security browsers typically receive weekly updates, and sometimes more. If you took over SMT apps what's to say you wouldn't abandon them as well?

**Comment by LearningAsIGo71** [comment]
> The general UX is getting worse for both users and devs for quite a few years now.

In what way is Android getting worse for users? I see nothing but improvements using GrapheneOS.

**Comment by Ponchale** [comment]
> > @tibbi Hello, a pleasure Alfonso, we are in charge of the Midori browser oriented towards privacy, security and before selling, please let's talk.
> 
> You are in charge of an abandoned app? Midori hasn't been updated on F-Droid in 2 years! To maintain proper security browsers typically receive weekly updates, and sometimes more. If you took over SMT apps what's to say you wouldn't abandon them as well?

Friend Midori has never been abandoned, the project is more alive than ever, we have been working on the migration to Gecko y Geckoview

https://news.itsfoss.com/midori-11/

https://ubunlog.com/midori-11-ya-fue-liberado-y-estas-son-sus-novedades/

https://alternativeto.net/news/2023/11/midori-11-0-released-with-improved-performance-and-integration-with-other-astian-services/

Midori, a project that was left for dead years ago, we achieved that and much more, the fact that in F-droid there is no update does not mean that it has been abandoned, the last update on Google Play is recent and we have not released anything because we are working now on Geckoview, and to the Midori community on Telegram we promised to return to F-droid but it has never been abandoned. 
![IMG_20231203_203043_580](https://github.com/SimpleMobileTools/General-Discussion/assets/285103/4e231f60-3e4e-42b9-a7a6-f6495dd90eda)

Please avoid uninformed comments or hateful messages or destructive criticism because we know what we're doing.

**Comment by tibbi** [comment]
While I appreciate everyones contribution, you should know that like 99% of the current code has been written by me and other paid devs, so no need to overreact the licensing thing...

**Comment by tibbi** [comment]
Anyway, thanks for the support, but Im locking the conversation as it doesnt really lead anywhere. Wishing you all the best :)
